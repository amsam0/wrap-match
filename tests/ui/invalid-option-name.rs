fn main() {}

#[wrap_match::wrap_match(option = 1)]
fn invalid_option_name() -> Result<(), CustomError> {
    Err(CustomError::Error)?;
    Ok(())
}
