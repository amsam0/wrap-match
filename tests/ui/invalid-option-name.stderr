error: wrap_match: unknown configuration option (expected `success_message`, `error_message`, `error_message_without_info`, `log_success` or `disregard_result`)
 --> tests/ui/invalid-option-name.rs:3:26
  |
3 | #[wrap_match::wrap_match(option = 1)]
  |                          ^^^^^^
